modules/sdk-coin-atom/src/lib/utils.ts

```diff
@@ -166,13 +193,21 @@ export class Utils implements BaseUtils {
     return encodedMessage.typeUrl;
   }
 
-  getGasBudgetFromDecodedTx(decodedTx: DecodedTxRaw): GasFeeLimitData {
+  getGasBudgetFromDecodedTx(decodedTx: DecodedTxRaw): FeeData {
```

Comment:
Kindly add jsdocs, lets be consistent with rest of the functions

Reply:
done

Reply:
Can you a decorate the params with type also and mention the return type too, Kindly do that for the rest of the functions also if you can